includes/class-sensei-lesson.php

```diff
@@ -2557,23 +2561,38 @@ public function lesson_add_existing_questions() {
 
 				foreach ( $questions as $question_id ) {
 
-					++$question_count;
-
 					$quizzes = get_post_meta( $question_id, '_quiz_id', false );
+
 					if ( ! in_array( $quiz_id, $quizzes ) ) {
+
+						$at_least_one_question_added = true;
+						$number_of_added_questions++;
```

Comment:
I don't think we need either of these new variables. See other comments.

Reply:
Please see my comment below! (In the main thread...)